### DISCUSSION TITLE
Aggiornamento

### DISCUSSION THREAD

**Discussion by Dregi56** [indomus/forum#545] [HUB: Home Assistant]
![Cattura](https://user-images.githubusercontent.com/79653938/126597226-8f19c954-ee25-4b47-a45a-9aa6e3c4d52a.JPG)
Leggo spesso nel blog di problemi con l' aggiornamento con difficoltà al successivo riavvio. La cosa mi mette una certa ansia....
Avete notizie relative a difficoltà con questa versione? Cosa dite, mi avventuro?

**Comment by KC-inDomus** [comment]
beh, se leggi le release notes capisci da solo se la tua configurazione può avere eventualmente problemi con l'aggiornamento - e come risolvere.
nello specifico, è una minor, quindi a logica dovresti poterlo passare senza problemi.

**Reply by Dregi56** [reply]
Grazie.

**Reply by nromagna** [reply]
Sinceramente (certo lo uso da solo 9 mesi) ma non ho mai avuto problemi di aggiornamento...

**Reply by Dregi56** [reply]
Fatto. Tutto OK! Grazie

**Reply by snakuzzo** [reply]
Bene...però ti ripeto quanto scritto già diverse volte nelle scorse settimane/mesi.
Non puoi aprire una discussione ogni volta per queste banalità....perché hai letto in giro di possibili problemi e perché ti viene l'ansia. Abbiamo anche una chat Telegram (te l'avevo già detto)

**Reply by indomus** [reply]
Dregi56 per favore limita gli interventi come indicato anche da Diego. Grazie
